HPDP  and MFHS nades

Magicocity1 opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
Add the HPDP nade and adds the metal foam grenade someone else made from #6956 to req
## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Parity 
## Technical details
<!-- Summary of code changes for easier review. -->
Yaml -- adds a new grenade based off the data I could find (not much -- smoke lifetime is 10s, grenade detonates in 3s, but not sure about the fire stats when someone is ignited).

MFHS appears to be the same as the custom metal foam grenade in cm13 (I could very well be wrong), so the MFHS just parents the custom metal foam and points at specific sprite (this is why unchanged is in the mfhs.rsi I didn't know how to remove that map in yaml...).
## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc).
Small fixes/refactors are exempt. Media may be used in RMC14 progress reports with credit. -->

https://github.com/user-attachments/assets/86c2975b-566e-4516-9419-b8ab3e880a1f

<img width="429" height="293" alt="image" src="https://github.com/user-attachments/assets/53b5d9fa-7e26-4cd2-926a-f21eb37e0277" />

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->
- [X] By submitting this code and/or assets, I confirm that I either own them or have provided the correct necessary licenses to use and distribute them. I agree to be fully responsible for any legal claims or issues arising from the use of these materials.

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Admin changes may be listed for admins to see with admin:
Coding changes with no changes visible in-game may be listed for other contributors with code:
Make sure to read the guidelines.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
:cl:
- add: Adds the HPDP nade to req (it's a smoke grenade but if you touch the smoke you get set on fire)
- add: The metal foam grenade from #6956 is added to req

Commit:
HPDP nade and add the metal foam someone else made to req

noctyrnal reviewed (commented):
If you search the repo theres alot of instances of TODOs for White Phosphorus grenades, I would really appreciate it if you could replace those TODOs with the grenade, and maybe missing packets and stuff too for it.

Magicocity1 committed:
hpdp packet sprites

Magicocity1 committed:
Update meta.json

for hpdp packet sprites

Magicocity1 committed:
Update meta.json

someone beat me to the hpdp stuffs and I didn't check

that's why measure twice cut once exists

QuietlyWhisper reviewed (commented):
sprites fine

Commit:
alright everything should be good now

Magicocity1 committed:
Merge branch 'RMC-14:master' into hpdp-mfhs

Commit:
ok merge conflicts should be over

Commit:
change custom metal foam to MFHS

Magicocity1 commented:
Gonna close it seems like the other white phos PR got merged

I'll break the metal foam into it's own PR
